### ISSUE TITLE
Suggestion: Add a new flag to terminate the program if one of the accounts is suspended

### ISSUE THREAD

**Issue by taisport** [farshadz1997/Microsoft-Rewards-bot#377]
Dear Farshadz,

I don't know why my accounts were suspended simultaneously. After checking the log, I noticed that the issue occurred after some MSN failed to execute. Therefore, I would like to suggest adding a flag to terminate the whole program immediately if one of the accounts is suspended. This could prevent the remaining accounts from being suspended.

![image](https://github.com/farshadz1997/Microsoft-Rewards-bot/assets/32865122/004142f2-3807-4608-80b2-73521b2e3b4f)

**Comment by yannikbaranski** [comment]
what was your froud score at that time?

**Comment by taisport** [comment]
Reply to yannikbaranski, around >30000 points I have lost.

**Comment by yannikbaranski** [comment]
No I mean your fraud score, that you can check here:
[https://scamalytics.com/](https://scamalytics.com/
)

**Comment by taisport** [comment]
Fraud Score: 29

**Comment by farshadz1997** [comment]
there's a ban wave happening after Microsoft figured out how some farms in 2 regions.

**Comment by taisport** [comment]
Yes, we can add a new argument to terminate the program immediately if one of the accounts is suspended. This will ensure that the remaining accounts are not suspended as a result.
